packages/block-library/src/page-list/block.json

```diff
@@ -0,0 +1,20 @@
+{
+	"apiVersion": 2,
+	"name": "core/page-list",
+	"category": "widgets",
+	"usesContext": [
+		"textColor",
+		"customTextColor",
+		"backgroundColor",
+		"customBackgroundColor",
+		"fontSize",
+		"customFontSize",
+		"showSubmenuIcon"
+	],
+	"supports": {
+		"reusable": false,
+		"html": false
+	},
+	"editorStyle": "wp-block-page-list-editor",
+	"style": "wp-block-page-list"
+}
```

Comment:
Should probably add a new line at the end. Other block.json files seem to have one.